docs/getting-started.md

```diff
@@ -183,7 +183,7 @@ yarn render:js
 All these commands can be executed at once passing the generated files through prettier:
 
 ```bash
-yarn render
+yarn render-php-js
```

Comment:
I was not able to run `render-php-js`
```
Script php scripts/render.php handling the render event returned with error code 255
error Command failed with exit code 255.
```
But by running first `start:php-ec` then I was able to `render-php-js`, build dist files and publish to gh-pages. 

Also `error Command "render:setupPhp" not found`, maybe documentation should be updated accordingly.

Reply:
I see in TWIG-180 that the correct cmd is now `setupPhp`